homeassistant/components/abode/binary_sensor.py

```diff
@@ -38,4 +41,6 @@ def is_on(self):
     @property
     def device_class(self):
         """Return the class of the binary sensor."""
+        if self._device.get_value("is_window") == "1":
```

Comment:
Does the _device api really return the string "1" ?

Can this be
`if self._device.get_value("is_window"):`
?

Reply:
Yeah it returns a string "1". I could also just add this as a property to the `AbodeBinarySensor` class in abodepy, so it could be something like: `if self._device.is_window:`

Reply:
`if self._device.get_value("is_window"):` will work since an empty string is a falsy value in python, e.g. try out `bool("")` which returns `False`

Reply:
```suggestion
        if self._device.get_value("is_window"):
```